docs-ui/components/userListElement.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withInfo} from '@storybook/addon-info';
+
+import UserListElement from 'sentry-ui/userListElement';
```

Comment:
imo because they are so similar, this would be better if we combined this into `<UserBadge>`, instead of having 2 different components.

Reply:
here is why i didn't do that (though tldr; I'm open minded):

1. UserBadge is multi-line so the html is pretty different
2. UserListElement does not have a link (but can be wrapped in one if u please) whereas UserBadge has one in the username, so the behavior is also different

If we combined them, the component would need to return a mostly different template based on some arbitrary prop like `isOneLine` or something. To me, it seemed better to start with two components which exhibit more consistent behavior. 

BUT I went back and forth on this so I'm clearly not 100% sold lol, and I'm sure there is plenty of  prior stuff on this so I'm basically down to do whatever is right.

Reply:
(This is not blocking for this PR, regardless of what we decide, let's handle in new follow-up PR)

I don't think template should determine whether it should be a different component or not. Functionally they are very similar and they even accept the same argument (`user`). I think having a prop is pretty much what we want and expect. `UserBadge` describes both of these components pretty well imo and they just have a few differences in how they are rendered (link, inline, showEmail, showUsername, etc).

Also, given our giant component library, less components is better.

Reply:
I'm in. The fact that they take the same props is pretty compelling.